### DISCUSSION TITLE
At start of the song playback speeds up for a couple of seconds then goes back to normal

### DISCUSSION THREAD

**Discussion by Dzonkins** [codetheweb/muse#1034] [Q&A]
I setup muse in docker container with all of the necessary paramteres needed in config but find little annoyance for every song I play it starts normal but after a second or 2 it speeds up for couple of seconds and then goes back to normal. What could be issue of this since machine I host it on is 16gb ram 6 core with lot of other services running

**Comment by codetheweb** [comment]
Hi, is this still happening?

**Reply by Dzonkins** [reply]
Seems like giving one more core to the main vm seemed to fix the issue, but another one I noticed is sometimes when I start to play big number of songs bot wont start new song after couple of songs and i need to enter rest of songs again for them to play
